test/test_autograd.py

```diff
@@ -6510,6 +6510,16 @@ def test_strided_leaf_grad_layout(self, device):
         (c * d).sum().backward()
         self.assertEqual(c.grad.stride(), (2, 1))
 
+    def test_moveaxis(self, device):
+        x = torch.randn(4, 3, 2, 1, dtype=torch.double, requires_grad=True)
```

Comment:
`device` isn't being used -- it should be passed to `torch.randn` so that we test that moveaxis works on both CPU and CUDA

Reply:
Great catch. Thanks!